File: mds/candile.py

import json


class Candile:
    def __init__(self, date= '', high= '', open= '', close= '', low= '', volume= ''):
        self.date  = date
        self.high  = high
        self.open  = open
        self.close  = close
        self.low  = low
        self.volume  = volume
    def toJSON(self) -> str:
        return json.dumps({self.date: {
            'high': self.high,
            'open': self.open,
            'close': self.close,
            'low': self.low,
            'volume': self.volume
        }})

File: sms/src/main/java/Candile.java
public class Candile {
    private int date;
    private double high;
    private double open;
    private double close;
    private double low;
    private double volume;

    public Candile(int date, double high, double open, double close, double low, double volume) {
        this.date = date;
        this.high = high;
        this.open = open;
        this.close = close;
        this.low = low;
        this.volume = volume;
    }

    public int getDate() {
        return date;
    }

    public double getHigh() {
        return high;
    }

    public double getOpen() {
        return open;
    }

    public double getClose() {
        return close;
    }

    public double getLow() {
        return low;
    }

    public double getVolume() {
        return volume;
    }
}
